### DISCUSSION TITLE
Support examples in package.json

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [npm/feedback#314] [Registry]
What if in `package.json` for an NPM package we could add an example:
```json
{
  //All the other stuff normally here.
  "examples": [
      "const express = require('express');\nconst app = express(); \napp.get('/', (req,res) => res.send('Hello express'))"
  ]
}
```
Then in Runkit from the NPM example page it'd load up with the example script! Maybe another tab in NPM called examples?

Tell me what you guys think!

**Comment by ljharb** [comment]
This is already possible with the `runkitExample` or `runkitExampleFilename` fields. See the [blog post](https://blog.runkit.com/2015/10/28/npm-plus-runkit/).

**Reply by Explosion-Scratch** [reply]
Really!?! Thanks!!!
